packages/knip/src/plugins/yarn/index.ts

```diff
@@ -0,0 +1,19 @@
+import type { IsPluginEnabled, Plugin } from '../../types/config.js';
+import { _glob } from '../../util/glob.js';
+
+// https://yarnpkg.com/features/constraints
+
+const title = 'yarn';
```

Comment:
The only nitpick I have is that Yarn seems to be the official spelling (title cased).